projects/angular-auth-oidc-client/src/lib/oidc.security.service.ts

```diff
@@ -61,6 +61,15 @@ export class OidcSecurityService {
     return this.callbackService.stsCallback$;
   }
 
+  /**
+   * Emits false when checkAuth observable emits a value, or errors. Initial value: true.
+   */
+  get isLoading$(): Observable<boolean> {
+    return this.isLoadingBehaviorSubject.asObservable();
```

Comment:
Please change the name in `isLoading` instead of `isLoadingBehaviorSubject`. The type should not be visible in the name of the variable.

Reply:
I have fixed this now.